features/symbol.yml

```diff
@@ -0,0 +1,28 @@
+name: Symbol
+description: The `Symbol` type represents a unique identifier.
```

Comment:
Heh, super abstract concept, but seems like this could use either more descriptiveness or an example, or both.